doc(encrypt): Allow customization of the cipher (#623)
longhorn/website: content/docs/1.4.0/advanced-resources/security/volume-encryption.md

Example secret your encryption keys are specified as part of the `CRYPTO_KEY_VALUE` parameter.
We use `stringData` as type here so we don't have to base64 encoded before submitting the secret via `kubectl create`.

> Parameters `CRYPTO_KEY_CIPHER`, `CRYPTO_KEY_HASH`, `CRYPTO_KEY_SIZE` and `CRYPTO_PBKDF` provide the customization for volume encryption.

Reviewer: ```suggestion
Besides `CRYPTO_KEY_VALUE`, parameters `CRYPTO_KEY_CIPHER`, `CRYPTO_KEY_HASH`, `CRYPTO_KEY_SIZE`, and `CRYPTO_PBKDF` provide the customization for volume encryption.
```

Author: Modified. Thanks.

> - `CRYPTO_KEY_CIPHER`: Set the cipher specification algorithm string. Default value is `aes-xts-plain64` for LUKS.

Reviewer: ```suggestion
- `CRYPTO_KEY_CIPHER`: Sets the cipher specification algorithm string. The default value is `aes-xts-plain64` for LUKS.
```

> - `CRYPTO_KEY_HASH`: Specifies the passphrase hash for `open`. Default value is `sha256`.

Reviewer: ```suggestion
- `CRYPTO_KEY_HASH`: Specifies the passphrase hash for `open`. The default value is `sha256`.
```

Author: Modified. Thanks.

> - `CRYPTO_KEY_SIZE`: Sets key size in bits and it must be a multiple of 8. Default value is `256`.

Reviewer: ```suggestion
- `CRYPTO_KEY_SIZE`: Sets the key size in bits and it must be a multiple of 8. The default value is `256`.
```

Author: Modified. Thanks.

> - `CRYPTO_PBKDF`: Set Password-Based Key Derivation Function (PBKDF) algorithm for LUKS keyslot. Default value is `argon2i`.

Reviewer: ```suggestion
- `CRYPTO_PBKDF`: Sets Password-Based Key Derivation Function (PBKDF) algorithm for LUKS keyslot. The default value is `argon2i`.
```

Author: Modified. Thanks.
